tests/unit/test_jobs/test_worker.py

```
@@ -0,0 +1,160 @@
+class TestProcessFileUpload:
+    """Tests for process_file_upload."""
+
+    @pytest.mark.unit
+    @pytest.mark.asyncio
+    async def test_returns_completed_result(self) -> None:
+        """Verify file upload processing returns completed status."""
+        from fragrance_rater.jobs.worker import process_file_upload
+
+        with patch("asyncio.sleep", new_callable=AsyncMock):
+            result = await process_file_upload(
+                {}, file_id="file-1", file_path="/tmp/test.csv"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.csv"

(S108)